test/validations/coercions_test.rb

```diff
@@ -22,4 +22,20 @@
     result = Lotus::Validations::Coercions.coerce(Integer, '')
     result.must_equal nil
   end
+
+  it 'returns an empty string when an empty string is given to coerce to String' do
+    result = Lotus::Validations::Coercions.coerce(String, '')
+    result.must_equal ''
+  end
+
+  it 'returns nil when an array is given to coerce to a string' do
+    result = Lotus::Validations::Coercions.coerce(String, [])
+    # This is not ideal behaviour
+    result.must_equal '[]'
```

Comment:
In terms of accepting untrusted input from the web, if the app receives a `[]` when it expects a string it's probably due to a malicious request. That's why I prefer the behaviour in Virtus:

```ruby
irb(main):001:0> SignUpForm.new(username: [])
Virtus::CoercionError: Failed to coerce attribute `username' from [] into String
        from /opt/rubies/2.1.5/lib/ruby/gems/2.1.0/gems/virtus-1.0.3/lib/virtus/attribute/strict.rb:19:in `coerce'
        from /opt/rubies/2.1.5/lib/ruby/gems/2.1.0/gems/virtus-1.0.3/lib/virtus/attribute/coercible.rb:14:in `set'
```